etna/analysis/plotters.py

```diff
@@ -1099,13 +1102,37 @@ def plot_trend(
         ax[i].legend()
 
 
+def get_fictitious_pvalues(pvalues: pd.DataFrame, alpha: float) -> Tuple[np.ndarray, float]:
+    """
+    Convert p-values into fictitious variables, the sum of which is 100.
+    Also converts alpha into fictitious variable.
+
+    Parameters
+    ----------
+    pvalues:
+        dataFrame with pvalues
+    alpha:
+        significance level, default alpha = 0.05
+    """
+    _, pvalues, _, _ = multipletests(pvals=pvalues, alpha=alpha, method="holm")
+    eps = 1  # magic constant
+    pvalues = np.array(pvalues)
+    pvalues = pvalues + eps
+    coef = 100 / sum(np.array(pvalues))
+    pvalues = coef * pvalues
+    new_alpha = coef * (alpha + eps)
+    return pvalues, new_alpha
+
+
 def plot_feature_relevance(
     ts: "TSDataset",
-    relevance_table: RelevanceTable,
+    relevance_table: Union[RelevanceTable, StatisticsRelevanceTable],
+    mode: Union[Literal["relevance"], Literal["p-values"]],
```

Comment:
Do we really need this parameter? According to the code the mode is determined by the type of `relevance_table`.